.gitmodules

```diff
@@ -6,10 +6,10 @@
 	url = https://github.com/paparazzi/luftboot.git
 [submodule "sw/ext/fatfs"]
 	path = sw/ext/fatfs
-	url = https://github.com/enacuavlab/fatfs.git
+	url = https://github.com/paparazzi/fatfs.git
 [submodule "sw/ext/chibios"]
 	path = sw/ext/chibios
-	url = https://github.com/ChibiOS/ChibiOS.git
+	url = https://github.com/paparazzi/ChibiOS.git
```

Comment:
There are only 2 minor commits compared to the ChibiOS official SVN. For the FDCAN I have supplied a patch, but that seems to be going to get fixed soonish. For the SPIv2 which is causing problems I currently don't have a fix and switched back all processors to SPIv1(I also don't know what is going wrong or why).
Both should eventually be fixed in the ChibiOS official SVN, this is only sort of needed temporary because I prefer to merge this rather sooner than later.